src/bots/bot.ts

```diff
@@ -268,9 +268,9 @@ export default abstract class BotClient {
     const channels = DataManager.getSubscriberData()[this.name];
 
     let gameSubs: Game[] = [];
-    let prefix = ``;
-    let label = ``;
-    let disabled = false;
+    let prefix: string | undefined = '';
+    let label: string | undefined = '';
+    let disabled: boolean | undefined = false;
```

Comment:
Weird that these can be undefined since they are initialized, isn't it?

Reply:
These values are all optional, `undefined` usually means that the default value is used

Reply:
why would we overwrite `''` with `undefined`? shouldn't we initialize with undefined instead?

Reply:
Fixed